integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/receive/EDUCoreConverter.java

```diff
@@ -39,10 +40,12 @@
         try {
             Marshaller marshaller = jaxbContext.createMarshaller();
             marshaller.marshal(new JAXBElement<>(new QName("uri", "local"), EDUCore.class, message), os);
-            message.setPayload(payloadConverter.unmarshallFrom(payloadAsString.getBytes()));
+            message.setPayload(payloadConverter.unmarshallFrom(payloadAsString.getBytes("UTF-8")));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "UTF-8" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1192)